### DISCUSSION TITLE
After deploy nestjs there give "This Serverless Function has crashed".

### DISCUSSION THREAD

**Discussion by mahabubr** [vercel/community#7376] [Help]
### Summary

i want to deploy my nestjs i got "This Serverless Function has crashed" on there. but in the logs there not have any error to show. and the deploy option there not have nestjs. so how can deploy nestjs on vercel,

### Steps to Reproduce

![Screenshot from 2024-07-05 21-58-40](https://github.com/vercel/community/assets/87204781/1aa874fa-4598-423b-9530-5ee7fd7f576d)

**Comment by versecafe** [comment]
You aren't getting logs as it fails during invocation, throw a try catch around the whole thing to get some logs out to start

https://vercel.com/docs/errors/FUNCTION_INVOCATION_FAILED

[a comment by an automated account is not shown]
